prometheus-exporters/aci-suite/charts/aci-correlator/values.yaml

```
@@ -5,7 +5,7 @@ replicaCount: 1
 
 image:
   name: aci-correlator
-  tag: "latest"
+  tag: DEFINED-IN-PIPELINE
```

```suggestion
  tag:
```

Leave it empty and add it to the test values please. If for some reason the pipeline can not replace it, it will otherwise end up in pulling the tag `DEFINED-IN-PIPELINE`.
You can add a comment above of course, which indicates where the tag is coming from.

---

done